defaults/preferences/zotero.js

```diff
@@ -214,6 +214,7 @@ pref("extensions.zotero.scaffold.eslint.enabled", true);
 
 // Tabs
 pref("extensions.zotero.tabs.title.reader", "titleCreatorYear");
+pref("extensions.zotero.tabs.reopenOnRestart", true);
```

Comment:
I'm realizing that this is a little more complicated. While we don't currently reopen reader or note windows (or, in the future, additional library windows) on restart, we obviously should, and the same pref should probably apply. So something like `extensions.zotero.restoreState` is probably better, moved up above `reopenPanesOnRestart`.

Reply:
Makes sense. I did that, and about reopening reader windows ...
I noticed that the current codebase has everything in place for reopening reader windows just like tabs. The data is saved in session.js, and on` init()` in reader.js, we check if there are saved reader windows and try to reopen them. So it seems like the reader windows _should_ be reopened, but they are not.

The issue is that `getSecondViewState` in reader.js is throwing an exception "this._iframeWindow.wrappedJSObject.getSecondViewState is not a function", which means that `Zotero.Reader.getWindowStates()` was throwing an exception which meant that reader updates were never saved by  `Zotero.Session.save()`. I will try to see if I can get to the root of the error but to begin, I wrapped `getSecondViewState` in `try-catch`, and it makes it so that reader windows are saved and reopened just like tabs.

Reply:
The notes windows don't seem to be saved in in Zotero.Session yet. Is that something I should try to add to this PR?

Reply:
> I noticed that the current codebase has everything in place for reopening reader windows just like tabs.

Oh, yeah, that's actually just a regression — it works in Z6. I opened https://github.com/zotero/zotero/issues/3376 for you.

> The notes windows don't seem to be saved in in Zotero.Session yet. Is that something I should try to add to this PR?

Sure! Then we could actually make a visible pref for restoring state.

Reply:
> Then we could actually make a visible pref for restoring state.

So, now the reader windows reopening should work the same as in Z6. Plus, I have the same logic working for notes opened in windows. All of that happens based on the `restoreState` preference. 

By visible pref, you mean ones in Zotero > Settings (on Mac)? Where should we place it? Maybe under General > Miscellaneous?